### ISSUE TITLE
Milestone 05 CAD

### ISSUE THREAD

**Issue by culbert-alexander** [culbert-alexander/LED_LIGHT#9]
The mechanical casing should be at a good starting point. I was unable to upload the entire Fusion 360 project due to the 9V battery component I imported, but I have STLs and .f3d files for my lid and main-base. I plan on having some simple screw to connect the PCB to the case and the lid to the main-base.

**Comment by ps178** [comment]
Can you move the CAD files under the folder Mechanical as instructed for milestone 05?

I think your CAD looks good! It might be too tall? But you can adjust that when you have the physical PCB, etc.

**Comment by culbert-alexander** [comment]
Yup that's on me, my bad! The CAD files are now under Mechanical.
And the longest dimension is 3 inches, I think the picture makes it look a lot longer than it really is!
